packages/web-components/src/divider/divider.stories.ts

```diff
@@ -0,0 +1,152 @@
+        fill="currentColor"
+      />
+    </svg>
+  </fluent-divider>
+`;
+
+export default {
+  title: 'Components/Divider',
+  args: {
+    content: 'Section One',
+    alignContent: 'center',
+    role: 'separator',
+    inset: false,
+    orientation: 'horizontal',
+  },
+  argTypes: {
+    content: {
+      description: 'HTML element wrapping text  (e.g. `<h3>Section One</h3>`), Image or SVG',
+      table: {
+        defaultValue: {
+          summary: 'empty',
+        },
+      },
+    },
+    alignContent: {
+      description: 'Align content',
+      table: {
+        type: {
+          summary: 'Fluent v9. Determines the alignment of the content within the divider.',
+        },
+        defaultValue: {
+          summary: 'center',
+        },
+      },
+      options: Object.values(DividerAlignContent),
+      control: {
+        type: 'select',
+      },
+    },
+    appearance: {
+      description: 'Divider and text colors',
+      table: {
+        type: {
+          summary: 'Fluent v9. A divider can have one of the preset appearances.',
+        },
+        defaultValue: {
+          summary: 'default',
+        },
+      },
+      options: Object.values(DividerAppearance),
+      control: {
+        type: 'select',
+      },
+    },
+    role: {
+      description: 'Set role attribute',
+      table: {
+        type: {
+          summary: 'Inherited from FASTDivider. Aria role for the divider.',
+        },
+        defaultValue: {
+          summary: 'separator',
+        },
+      },
+      options: Object.values(DividerRole),
+      control: {
+        type: 'select',
+      },
+    },
+    inset: {
+      description: 'Pad the ends of divider',
+      table: {
+        type: {
+          summary:
+            'Type: boolean. Fluent v9. Divider layout is block for strict distinctions between items, or inset for closer relationships with neighboring content.',
+        },
+        defaultValue: {
+          summary: false,
+        },
+      },
+      control: 'boolean',
+    },
```

Comment:
I'm not sure of the function of all these variations in this object, I'd like @miroslavstastny to validate that these all make sense, etcc.

Reply:
This adds a bunch of information to the attributes table: https://fluentuipr.z22.web.core.windows.net/pull/26901/web-components/storybook/index.html?path=/docs/components-divider--divider.
We have not agreed on the content which should go there, so I would suggest to go with whatever is in the PR and refactor once we agree.